WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeBrowserAdapter.java

```diff
@@ -150,26 +178,26 @@ public void bindView(View view, Context context, Cursor cursor) {
             screenshotURL = ThemeWebActivity.THEME_HTTPS_PROTOCOL + screenshotURL;
         }
 
-        configureImageView(themeViewHolder, screenshotURL, themeId, isCurrent);
-        configureImageButton(context, themeViewHolder, themeId, isPremium, isCurrent);
-        configureCardView(context, themeViewHolder, isCurrent);
-    }
+        configureImageView(holder, screenshotURL, themeId, isCurrent);
+        configureImageButton(holder, themeId, isPremium, isCurrent);
+        configureCardView(holder, isCurrent);
 
-    @Override
-    public int getItemViewType(int position) {
-        Cursor cursor = (Cursor) getItem(position);
-        return getItemViewType(cursor);
-    }
+        // show a section header if one exists at this position
+        ThemeSectionHeader header = mHeaders.get(position);
+        if (header != null) {
+            holder.headerView.setVisibility(View.VISIBLE);
+            holder.headerText.setText(header.text);
+            holder.headerCount.setText(Integer.toString(header.count));
```

Comment:
We can use `String.valueOf(header.count)` to get rid of the AS localization warning here.

Reply:
@oguzkocer I just addressed both of these in a0f796f